### PR TITLE
Added prowl notifier

### PR DESCRIPTION
**Pull request by bryanberger**
Added support for Prowl push notifications.
https://www.prowlapp.com/

### CODE REVIEW

**Review by DeviaVir — CHANGES_REQUESTED** [review]
you'll also need to update and commit the `package-lock.json`:
```
rm -rf package-lock.json && npm i && git add package-lock.json && git commit && git push
```

File: extensions/notifiers/prowl.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var Prowl = require('node-prowl')
+
+module.exports = function container (get, set, clear) {
+  var prowl = {
+    pushMessage: function(config, title, message) {
+      var p = new Prowl(config.key)
+      p.push(message, title, function(err, remaining) {
```
Comment (DeviaVir) [inline extensions/notifiers/prowl.js]: `remaining` is unused here?

**Comment by bryanberger** [comment]
@DeviaVir updated

**Review by DeviaVir — APPROVED** [review]

**Comment by DeviaVir** [comment]
Cool, thanks for contributing! :+1:

**Pull request merged**